pkg/config/config.go

```diff
@@ -28,6 +28,15 @@ type Config struct {
 	sshConfigPath string
 }
 
+// IncludedFiles returns the list of the included files
+func (c *Config) IncludedFiles() []string {
+	includedFiles := []string{}
+	for file, _ := range c.includedFiles {
```

Comment:
should omit 2nd value from range; this loop is equivalent to `for file := range ...`